compiler/target.jou

```diff
@@ -92,6 +94,10 @@ def init_global_target() -> None:
         strcpy(global_compiler_state.target.triple, triple)
         LLVMDisposeMessage(triple)
 
+    if (starts_with(global_compiler_state.target.triple, "armv6") and
+        ends_with(global_compiler_state.target.triple, "hf")):
+        target_feature_flags = "+vfp2"
+
```

Comment:
Please move this logic to `init_thread_local_target()` so that you don't need a global variable. (It will run several times, but that doesn't matter.)